zephyr/CMakeLists.txt

```diff
@@ -113,6 +113,11 @@ if (CONFIG_SOC_SERIES_INTEL_ADSP_BAYTRAIL)
 		${SOF_SRC_PATH}/schedule/dma_multi_chan_domain.c
 	)
 
+	# SOF core infrastructure - runs on top of Zephyr
+	zephyr_library_sources(
+		${SOF_SRC_PATH}/schedule/ll_schedule.c
+	)
+
```

Comment:
if you respin this PR, please merge this with the previous commit to avoid breaking bisection, even though these platforms don't actually work with Zephyr...

Reply:
Done